azure-pipelines/end-to-end-tests-dir/overlays.ps1

```diff
@@ -59,6 +60,11 @@ $manifestRoot = "$TestingRoot/overlays-malformed-shadowing"
 Run-Vcpkg install --x-manifest-root=$manifestRoot
 Throw-IfFailed
 
+# ... even if that subsequent port is the builtin root
+$manifestRoot = "$TestingRoot/overlays-malformed-shadowing-builtin"
+Run-Vcpkg install --x-manifest-root=$manifestRoot --x-builtin-ports-root "$manifestRoot/builtin-malformed"
```

Comment:
Before the fix 
![image](https://github.com/user-attachments/assets/f0a74ab2-841c-4714-bf6a-b0ac13a292f4)